command.go

```diff
@@ -49,12 +51,26 @@ func (c *Command) Run(variables map[string]string) {
 	var stderr bytes.Buffer
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
-	err := cmd.Run()
-	if err == nil {
-		log.Printf("Success %v", c.Name)
-	} else {
+	err := cmd.Start()
+	if err != nil {
 		log.Printf("Failed %v: %v", c.Name, err)
 		log.Print("stdout: " + stdout.String())
 		log.Print("stderr: " + stderr.String())
+
+	} else {
+		// wait or timeout
+		donec := make(chan error, 1)
+		go func() {
+			donec <- cmd.Wait()
+		}()
+		select {
+		case <-time.After(c.Timeout):
+			killcmd := exec.Command("taskkill", "/F", "/T", "/PID", strconv.Itoa(cmd.Process.Pid))
```

Comment:
Let's keep this platform-agnostic, by moving the taskkill execution into it's own file.

1. Create a new Go file called platform_windows.go. Inside that file, create a function called killProcessTree(pid int)
2. Create another Go file called platform_other.go. Mark that file as building on all platforms other than Windows, by using a build tag // +build !windows

Now, when we try to compile on non-windows, we'll get "function killProcessTree" not found, and it will be obvious that it needs to be implemented. Also, leave a comment above killProcessTree, telling us that it kills the process and all children. And also mention that this is necessary for killing git.